<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaCadastroApto</class>
 <widget class="QDialog" name="JanelaCadastroApto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>842</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>842</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro Apartamento</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bairro:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cidade">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="endereco">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Endereço:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorAluguel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Observações:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="piscina">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Piscina</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dataVencimento">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="displayFormat">
    <string>d/M</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor Aluguel:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome Locatário:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dormitórios:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locatario">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Vencimento Aluguel:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="observacoes">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bloco</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dormitorios">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="bairro">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bloco">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="imobiliaria">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>Selecione...</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Imobiliária 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Imobiliária 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Imobiliária 3</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="vagasGaragem">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vagas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Andar:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="andar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorCondominio">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor Condomínio:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="academia">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Academia</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cobertura">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Cobertura</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="salaodefestas">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Salão de Festas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAp">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>390</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Imobiliaria</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="complemento">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Complemento</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>endereco</tabstop>
  <tabstop>bloco</tabstop>
  <tabstop>bairro</tabstop>
  <tabstop>cidade</tabstop>
  <tabstop>locatario</tabstop>
  <tabstop>dormitorios</tabstop>
  <tabstop>vagasGaragem</tabstop>
  <tabstop>andar</tabstop>
  <tabstop>imobiliaria</tabstop>
  <tabstop>piscina</tabstop>
  <tabstop>dataVencimento</tabstop>
  <tabstop>valorAluguel</tabstop>
  <tabstop>valorCondominio</tabstop>
  <tabstop>observacoes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
